neuraxle/steps/loop.py

```diff
@@ -146,6 +161,36 @@ def should_resume(self, data_container: DataContainer, context: ExecutionContext
         return False
 
 
+class ContinueException(Exception):
+    pass
+
+
+class BreakException(Exception):
+    pass
+
+
+class Break(Identity):
```

Comment:
The break class could inherit from the mixin that makes the `transform` function fallback on `_handle_transform`. Something like a force handle mixin or like that. This is because you've put your code inside the `_did_process` so even the simplified lifecycle methods needs to call the did_process. 

E.g.: step.transform() should call the _did_process otherwise the condition will never be evaluated. 

Same for the Continue class.